bee-storage/bee-storage/README.md

```diff
@@ -0,0 +1 @@
+# bee-storage
```

Comment:
I suggest adding a small sentence like "Storage layer for the Bee framework.", or something similar.